test/spec/modules/sublimeBidAdapter_spec.js

```diff
@@ -0,0 +1,205 @@
+import { expect } from 'chai';
+import { spec } from 'modules/sublimeBidAdapter';
+import { newBidder } from 'src/adapters/bidderFactory';
+
+describe('Sublime Adapter', () => {
```

Comment:
Can you please update these arrow function references to the standard `function () {` style?  The arrow function syntax is discouraged for Mocha.

Reply:
done

Reply:
done